resources/views/pages/home-participant.blade.php

```diff
@@ -1,23 +1,80 @@
 @extends('master')
 
+@php 
+	use App\Helpers\DateHelper; 
+	use Carbon\Carbon;
+@endphp
+
 @section('title')
 	Home
 @endsection
 
 @section('content')
 
 <div class="jumbotron">
-	<h1>Welkom, {{ Auth::user()->profile->voornaam }}!</h1>
+	<h1>Welkom, {{ $participant->voornaam }}!</h1>
+
 	<p>
-		Leuk je weer te zien. Ga je weer een keer mee op kamp?
+		Leuk je weer te zien. 
+		@if ($participant->geboortedatum->isBirthday())
+			<b>Gefeliciteerd met je verjaardag!</b> <span class="glyphicon glyphicon-gift"></span>
+		@endif
 	</p>
-	@if ($congrats == 1)
+
+	@forelse ($registeredCamps as $c)
+	<div class="panel panel-default">
+		<div class="panel-body">
+			<h2>
+				Status inschrijving <a href="{{ url('events', $c->id) }}">{{ $c->naam }}</a> ({{ DateHelper::Format($c->datum_start) }} t/m {{ DateHelper::Format($c->datum_eind) }}):
+			</h2>
+			<hr/>
+			<p>
+				@if ($c->pivot->isPaid())
+					<span class="glyphicon glyphicon-ok" aria-hidden="true"></span> &nbsp;
+					Betaling ontvangen op {{ DateHelper::Format(new Carbon($c->pivot->datum_betaling)) }}.
+				@else
+					<span class="glyphicon glyphicon-warning-sign" aria-hidden="true"></span> &nbsp;
+					Betaling nog niet ontvangen! <a href="{{ url('pay', $c->id) }}">Betaal direct @money($c->pivot->getPayment()->getTotalAmount()) via iDeal.</a>
+				@endif
+			</p>
+			<p>
+				@if ($participant->inkomen == 0)
+					<span class="glyphicon glyphicon-ok" aria-hidden="true"></span> &nbsp;
+					Geen inkomensverklaring nodig (geen korting).
+				@elseif ($participant->inkomensverklaring !== null)
+					<span class="glyphicon glyphicon-ok" aria-hidden="true"></span> &nbsp;
+					Inkomensverklaring ontvangen op {{ DateHelper::Format($participant->inkomensverklaring) }}.
+				@else
+					<span class="glyphicon glyphicon-warning-sign" aria-hidden="true"></span> &nbsp;
+					Inkomensverklaring nog niet ontvangen! Stuur deze z.s.m. naar <a href="mailto:penningmeester@anderwijs.nl">penningmeester@anderwijs.nl</a>.
+				@endif
+			</p>
+			<p>
+				@if ($c->pivot->geplaatst)
+					<span class="glyphicon glyphicon-ok" aria-hidden="true"></span> &nbsp;
+					Geplaatst voor het kamp. Veel plezier!
+				@else
+					<span class="glyphicon glyphicon-info-sign" aria-hidden="true"></span> &nbsp;
+					Nog niet geplaatst &mdash; dit gebeurt ongeveer twee weken voor het kamp.
+				@endif
+			</p>
+		</div>
+		<div class="panel-footer">
+			<p><small>
+				Op onze website meer informatie over het <a href="https://anderwijs.nl/inschrijven/inschrijven-scholieren/" target="_blank">stappenplan voor inschrijving</a>. 
+			</small></p>
```

Comment:
not a fan of using <small> inside a <p> for styling purposes. Adding a bootstrap class like `small` is nicer. But I noticed that the Jumbotron styling is in the way, oh bootstrap 3. how annoying you are.

Reply:
Klopt ja, `p.small` heeft hier geen effect. Of ik moet een style override toevoegen maar dat is ook lelijk. Zou dit voor nu zo willen laten maar snap je punt.

Reply:
Ja ik probeerde al dingen te hannessen, maar alles was slechter dan dit. dus.. meh.